pom.xml

```diff
@@ -175,6 +175,14 @@
           </execution>
         </executions>
       </plugin>
+        <plugin>
```

Comment:
Sorry Peter, minor issue, but please fix it. There is some problem with the indent here. Please use the right number of spaces here.